wochenschau-verlag.csl

```diff
@@ -0,0 +1,301 @@
+                </group>
+              </if>
+            </choose>
+          </if>
+          <else-if type="article-journal">
+            <choose>
+              <if variable="DOI volume" match="none">
+                <text value="Online:" prefix=" " suffix=" "/>
+                <text variable="URL"/>
+                <group prefix=" (" suffix=")">
+                  <text value="Zugriff:" suffix=" "/>
+                  <date variable="accessed">
+                    <date-part name="day" form="numeric" suffix="."/>
+                    <date-part name="month" form="numeric" suffix="."/>
+                    <date-part name="year" form="long"/>
+                  </date>
+                </group>
+              </if>
+            </choose>
+          </else-if>
+          <else-if type="article-journal" variable="DOI" match="none">
+            <text value="Online:" prefix=" " suffix=" "/>
+            <text variable="URL"/>
+            <group prefix=" (" suffix=")">
+              <text value="Zugriff:" suffix=" "/>
+              <date variable="accessed">
+                <date-part name="day" form="numeric" suffix="."/>
+                <date-part name="month" form="numeric" suffix="."/>
+                <date-part name="year" form="long"/>
+              </date>
+            </group>
+          </else-if>
+        </choose>
+      </if>
+    </choose>
+  </macro>
+  <macro name="title">
+    <text variable="title" text-case="title"/>
+    <choose>
+      <if type="book" variable="volume" match="all">
+        <text variable="volume" form="short" prefix=", Bd. "/>
+      </if>
+      <else-if type="book" variable="number-of-volumes" match="all">
+        <text variable="number-of-volumes" prefix=", " suffix=" Bände"/>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="genre">
+    <choose>
+      <if type="report" match="any">
+        <group suffix=".">
+          <text variable="genre" suffix=" "/>
+          <text variable="number"/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="publisher">
+    <group delimiter=": ">
+      <text variable="publisher-place"/>
+    </group>
+  </macro>
+  <macro name="year-date">
+    <choose>
+      <if variable="issued">
+        <date variable="issued">
+          <date-part name="year"/>
+        </date>
+      </if>
+      <else>
+        <text term="no date" form="short"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="locator">
+    <choose>
+      <if type="article-journal">
+        <choose>
+          <if variable="volume" match="none">
+            <text variable="DOI" prefix="DOI: "/>
+          </if>
+          <else>
+            <group delimiter=" ">
+              <text variable="volume"/>
+              <text variable="issue" prefix="(" suffix=")"/>
+            </group>
+          </else>
+        </choose>
+      </if>
+    </choose>
+  </macro>
+  <macro name="locator-citation">
+    <group>
```

Comment:
this group does nothing. I'd also improve this macro to add the locator label if the locator is NOT a page.